perSystem/devShells/default.nix

```diff
@@ -0,0 +1,47 @@
+flake: {
+  perSystem = {
+    config,
+    pkgs,
+    system,
+    ...
+  }: let
+    jsonld = (import ../../jsonld-nix { inherit pkgs system; inherit (pkgs) nodejs; }).nodeDependencies;
+    votingScripts = pkgs.runCommandNoCC "voting-scripts" {} ''
+      mkdir -p $out/bin
+      cp ${../../scripts}/* $out/bin
+      substituteInPlace $out/bin/* --replace "source ./check-env" "source $out/bin/check-env"
+    '';
+  in {
+    cardano-parts.shell = {
+      global = {
+        defaultShell = "test";
+        enableVars = false;
+        defaultHooks = ''
+          # CURRENTLY BROKEN!
+          alias cardano-node=cardano-node-ng
+          alias cardano-cli=cardano-cli-ng
+        '';
+      };
+      test = {
+        enableVars = true;
+        defaultVars = {
+          CARDANO_NODE_SOCKET_PATH = "./cc-public/node.socket";
+          CARDANO_NODE_NETWORK_ID = "5";
```

Comment:
Demeter already injects these into the workspace environment, will this interfere with that if you drop into the devshell in the workspace? Now that jsonld is in the nix shell, that will become a necessary setup step.